pkg/authorization/api.go

```diff
@@ -65,6 +65,11 @@ type Request struct {
 	// RequestPeerCertificates stores the request's TLS peer certificates in PEM format
 	RequestPeerCertificates []*PeerCertificate `json:"RequestPeerCertificates,omitempty"`
 
+	// RemoteAddr stores the network address that sent the request
+	// There's many cases where the remote address may not be the actual
+	// remote address (e.g., proxy being used, or classic swarm, or the unix-socket being used)
+	RemoteAddr string `json:"RemoteAddr,omitempty"`
```

Comment:
Discussing with @anusha-ragunathan - perhaps this should be called `RequestAddr` (i.e., where the request comes from) ?

Reply:
field name has been changed.